Stellar-ledger-entries.x

```diff
@@ -518,20 +518,13 @@ case CONFIG_SETTING_TYPE_UINT32:
 
```

Comment:
Is the `ConfigSetting` still needed? I see the `ConfigSettingEntry` below is using `uint32` directly so maybe this is obsolete?

Reply:
Good catch, we indeed don't use this anymore.

Reply:
Yeah good catch. Removed.